### DISCUSSION TITLE
Another thinking diagram...

### DISCUSSION THREAD

**Discussion by trbouma** [trustoverip/tswg-trust-registry-tf#77] [General]
Another thinking diagram for consideration:

![Trust model   Trust Registry discussion slides](https://user-images.githubusercontent.com/12282252/220357564-d2520a35-dc69-44a9-a2cc-9c53c98a34c6.png)

The idea here is that the things in circles (intention, authority, norm, action) are ultimately human things that can never be fully-encoded. But what can be encoded is a rule (the box) - is a creation or combination of the four human things. The purpose of a rule is to effect an action (or decision), that is based on authority, intention and norm, none of which can be completely controlled.

Why is this relevant to the trust registry? It helps us to scope. Whatever we create will be encoded as rules (methods, etc) and will always be an imperfect embodiment of authority, intention, norm, and action.  The rules need to be as simple as possible to deal with this ongoing imperfection.

**Comment by IDmachines** [comment]
+1 wrt rules, e.g.  specifiy default state
